ConfigurationSystem/private/Modificator.py

```diff
@@ -77,202 +78,202 @@ def getDictRootedAt(path):
 
     return getDictRootedAt(root + "/" + relpath)
 
-  def getValue( self, optionPath ):
-    return gConfigurationData.extractOptionFromCFG( optionPath, self.cfgData )
+  def getValue(self, optionPath):
+    return gConfigurationData.extractOptionFromCFG(optionPath, self.cfgData)
 
-  def sortAlphabetically( self, path, ascending = True ):
-    cfg = self.__getParentCFG( path, parentLevel = 0 )
+  def sortAlphabetically(self, path, ascending=True):
+    cfg = self.__getParentCFG(path, parentLevel=0)
     if cfg:
-      if cfg.sortAlphabetically( ascending ):
-        self.__setCommiter( path )
+      if cfg.sortAlphabetically(ascending):
+        self.__setCommiter(path)
 
-  def __getParentCFG( self, path, parentLevel = 1 ):
-    sectionList = List.fromChar( path, "/" )
+  def __getParentCFG(self, path, parentLevel=1):
+    sectionList = List.fromChar(path, "/")
     cfg = self.cfgData
     try:
       if parentLevel > 0:
         sectionList = sectionList[:-parentLevel]
       for section in sectionList:
-        cfg = cfg[ section ]
+        cfg = cfg[section]
       return cfg
-    except:
+    except BaseException:
```

Comment:
This is what `autopep8` does

Reply:
It does that to avoid changing behaviour as using Exception is not exactly equivalent.

From PEP-8:

> A bare except: clause will catch SystemExit and KeyboardInterrupt exceptions, making it harder to interrupt a program with Control-C, and can disguise other problems. If you want to catch all exceptions that signal program errors, use except Exception: (bare except is equivalent to except BaseException:).

DIRAC already captures the signals directly so there will be few (if any) cases where capturing low level exceptions is intentional.

Reply:
It does that to avoid changing behaviour as using Exception is not exactly equivalent.

From PEP-8:

> A bare except: clause will catch SystemExit and KeyboardInterrupt exceptions, making it harder to interrupt a program with Control-C, and can disguise other problems. If you want to catch all exceptions that signal program errors, use except Exception: (bare except is equivalent to except BaseException:).

DIRAC already captures the signals directly so there will be few (if any) cases where capturing low level exceptions is intentional.